internal/api/v1/resource.go

```diff
@@ -0,0 +1,187 @@
+package v1
+
+import (
+	"context"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/homedepot/go-clouddriver/internal"
+	"github.com/homedepot/go-clouddriver/internal/kubernetes"
+	clouddriver "github.com/homedepot/go-clouddriver/pkg"
+	"github.com/jinzhu/gorm"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var (
+	// Only need the kinds that show in the infrastructure pages.
+	infrastructureKinds = []string{
+		"daemonSets",
+		"deployments",
+		"ingresses",
+		"pods",
+		"replicaSets",
+		"services",
+		"statefulSets",
+	}
```

Comment:
I don't think we need pods. The clusters page lists all pods for us anyways, and the main purpose of this should be to associate an account to an application, which you are doing by grabbing the application from the annotation. The clusters page really cares about everything else in the list. It's going to list pods on the `/applications/*/serverGroups` and `/applications/*/loadBalancers` endpoints for us anyways to associate pods accordingly. Additionally, the clusters page is going to list all pods and filter them on this annotation anyways. https://github.com/homedepot/go-clouddriver/blob/master/internal/api/core/applications.go#L1462

If I'm missing something, let me know.